packages/amplication-e2e-tester/tests/create.test.js

```diff
@@ -0,0 +1,50 @@
+const TITLE = "Amplication";
+const LOGIN_URL = "http://localhost:3001/login";
+const HOME_PAGE_URL = "http://localhost:3001/";
+const USER_EMAIL = "lisa@simpson.com";
+const USER_PASSWORD = "secret42";
+const APP_NAME = "my second app";
+const CONTINUE_BUTTON_CONNTENT = "Continue";
+const CREATE_BUTTON_CONTENT="Create App";
+const TIMEOUT = 60000;
+async function URL_VALIDITY(RESULT_URL,CORRECT_URL)
+{
+  await expect(RESULT_URL).toMatch(CORRECT_URL)
+};

```

Comment:
The format of this function is incorrect. Have you used Prettier to format the file?
Function parameters should be in camel case.